internal/factsengine/gatherers/verifypassword_test.go

```diff
@@ -88,8 +88,8 @@ func (suite *PasswordTestSuite) TestPasswordGatherNotEqual() {
 	suite.ElementsMatch(expectedResults, factResults)
 }
 
-func (suite *PasswordTestSuite) TestPasswordGatherCryptError() {
-	shadow := []byte("hacluster:$aaaa$aaaa")
+func (suite *PasswordTestSuite) TestPasswordGatherNotEncrypted() {
+	shadow := []byte("hacluster:!:19029::::::")
```

Comment:
Hey @arbulu89, just wondering about this: why is the test name "...NotEncrypted"? seems to me it's more simply about the case where no password is set (rather than not encrypted).

Also, about the exclamation mark, I think an exclamation mark at the start of the field means simply "account disabled" while an exclamation mark in the content of the hash is an invalid hash (as it is an invalid character in this context)

Reply:
Yes, we could improve the naming of the text.
About what the `!` means, based on this:

Reply:
I think I understand my doubts now, there is a "common practice" (even documented in `man 5 shadow`) that says that if you add a `!` as the first character of a password it means "password disabled" but in reality, adding it anywhere will block the password anyways. So effectively, a password containing just `!` or `*` means: "no password, and also no login allowed".